data/presets/tourism/camp_site.json

```diff
@@ -5,13 +5,15 @@
         "operator",
         "address",
         "access_simple",
-        "capacity",
         "fee",
         "payment_multi_fee",
         "charge_fee"
     ],
     "moreFields": [
         "backcountry",
+        "capacity:caravans",
```

Comment:
it depends a bit, but I think capacity:persons is the most important one...

Reply:
I just moved all 3 to the regular fields. It would eventually be nice to have a kind of "grouped fields" feature in iD (see https://github.com/openstreetmap/iD/issues/8104) for such fields.